### PR TITLE
added chapter about HashBundle

### PR DESCRIPTION
**Pull request by danrot**

### CODE REVIEW

**Comment by danrot** [comment]
@webmozart @wachterjohannes Can you please have a look, so that we can merge this?

File: bundles/hash.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
+HashBundle
+==========
+
+The SuluHashBundle adds an easy to use solution for protecting entities and
+documents from being overridden by accident.
```
Comment (webmozart) [inline bundles/hash.rst]: Could you add an example of where this could be the case?

File: bundles/hash.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Serialization
+-------------
+
+The `HashSerializeEventSubscriber` is the most important part of the
+HashBundle, since it is its job to add the hash of the object to the serialized
```
Comment (webmozart) [inline bundles/hash.rst]: "HashBundle. The subscriber's job is to add ..."

In general, try to keep sentences short. Use simple grammar. That's much easier to read. :)

File: bundles/hash.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+representation being delivered in the response. So the serializer has to be
+used to deliver the response with the hash value automatically.
+
+Hashing
+-------
+
+The `HashSerializeEventSubscriber` uses the `AuditableHasher` for hashing
+documents and entities. Therefore it simply uses the changer (a user) and the
```
Comment (webmozart) [inline bundles/hash.rst]: This hasher calculates a hash based on the user who modified a document and the time when that change happened.

**Comment by danrot** [comment]
@webmozart updated :smiley:

**Comment by chirimoya** [comment]
@danrot ... please add example how to handle the controller logic. else ACK

**Pull request merged**